src/test/java/com/sap/cloudfoundry/client/facade/util/CloudStackCacheTest.java

```diff
@@ -0,0 +1,61 @@
+package com.sap.cloudfoundry.client.facade.util;
+
+import com.sap.cloudfoundry.client.facade.domain.CloudStack;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.Mockito;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class CloudStackCacheTest {
```

Comment:
Since the cache is just basically a wrapper around a hashtable, is this test class _really_ needed?